instance/templates/instance/ansible/vars.yml

```diff
@@ -181,11 +181,33 @@ POSTFIX_QUEUE_SENDER_CANONICAL_MAPS: '{{ smtp_relay_settings.source_address }}
 {% endif %}
 {% endwith %}
 
+
+
+# EDXAPP_CMS_NGINX_PORT: 80
```

Comment:
Ah, this is cruft; I had copied it from above (where it isn't commented) and was using it as a mental template to build the vars for discovery and ecommerce. Will remove.